tracing-subscriber/src/fmt/format/pretty.rs

```diff
@@ -265,7 +285,11 @@ impl<'a> field::Visit for PrettyVisitor<'a> {
 
     fn record_error(&mut self, field: &Field, value: &(dyn std::error::Error + 'static)) {
         if let Some(source) = value.source() {
-            let bold = self.style.bold();
+            let bold = if self.ansi {
+                self.style.bold()
+            } else {
+                Style::new()
+            };
```

Comment:
nit, take it or leave it: we _could_ factor this out into an inherent method on `PrettyVisitor`, since it shows up a couple of times. not a blocker though, up to you!